README.md

```diff
@@ -1418,6 +1418,7 @@ Here's an example of what the notifications look like:
 | `alerting.telegram`                   | Configuration for alerts of type `telegram`                                                | `{}`                       |
 | `alerting.telegram.token`             | Telegram Bot Token                                                                         | Required `""`              |
 | `alerting.telegram.id`                | Telegram User ID                                                                           | Required `""`              |
+| `alerting.telegram.topic-id`          | Telegram Topic ID in Group                                                                 | `""`                       |    
```

Comment:
Can you specify that this also corresponds to `message_thread_id`?

Reply:
@TwiN done